{"name": "Запрос с несуществующим номером заказа возвращает ошибку", "status": "passed", "steps": [{"name": "Получение информации о заказе по трек-номеру", "status": "passed", "parameters": [{"name": "track_number", "value": "99999999"}], "start": 1735025507692, "stop": 1735025508585}], "start": 1735025507692, "stop": 1735025508586, "uuid": "2848f0d6-5862-4b89-a53f-523b6015e147", "historyId": "39d6d4ca54158ef7e4a76431cc3f205c", "testCaseId": "39d6d4ca54158ef7e4a76431cc3f205c", "fullName": "tests.orders.test_get_orders.TestOrders#test_get_order_by_track_nonexistent_number", "labels": [{"name": "feature", "value": "Заказы"}, {"name": "parentSuite", "value": "tests.orders"}, {"name": "suite", "value": "test_get_orders"}, {"name": "subSuite", "value": "TestOrders"}, {"name": "host", "value": "MBP-Dmitrij.lan"}, {"name": "thread", "value": "2456-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.orders.test_get_orders"}]}